### ISSUE TITLE
Add a readme

### ISSUE THREAD

**Issue by tomasmfc** [ddnet/ddnet-web#282]
Would be nice to have some basic information on this repo

**Comment by kaitlynia** [comment]
Agreed. Some information about the NGINX config and templating would be great so others can test changes before they are uploaded
